### DISCUSSION TITLE
Samsung Motion Photo

### DISCUSSION THREAD

**Discussion by wengtad** [immich-app/immich#2135] [Q&A]
Hi all, thanks for all the efforts and developed this great app. 

Just wanna ask is there a support for Android (Samsung) Motion Photo?
I know Live Photo for iOS is supported, just not sure about Android.

If it is supported, may I know how to do it? Because I had try enable motion photo on my phone and take a picture. In the camera gallery I'm able to view the motion photo.

So I try to sync to Immich, but it doesn't seem to work. It just showing normal jpg picture.

Attach the detail of the photo via camera gallery.

![Screenshot_20230331_114817_Gallery.jpg](https://user-images.githubusercontent.com/12437747/229018193-31fcd5d7-dd0c-4359-8ab1-de539e405d1f.jpg)

**Comment by alextran1502** [comment]
Hello Motion Photo is not yet supported in Immich

**Reply by wengtad** [reply]
Hello, ok noted, hope to have this feature soon.
